features/logging/cluster_log_forwarder.feature

```diff
@@ -295,3 +295,83 @@ Feature: cluster log forwarder features
       | file                                 | app_pipeline_name     | infra_pipeline_name   | audit_pipeline_name   |
       | clf-forward-with-same-tag.yaml       | forward-to-default-es | forward-to-default-es | forward-to-default-es | # @case_id OCP-33750
       | clf-forward-with-different-tags.yaml | forward-app-logs      | forward-infra-logs    | forward-audit-logs    | # @case_id OCP-33893
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33627
+  @admin
+  @destructive
+  Scenario: Forward logs to remote-syslog - config error
+    Given the master version >= "4.6"
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+
+    Given rsyslog receiver is deployed as insecure in the "openshift-logging" project
+    Given admin ensures "instance" cluster_log_forwarder is deleted from the "openshift-logging" project after scenario
+    Given I obtain test data file "logging/clusterlogforwarder/rsyslog/rsys_clf_invalid_values.yaml"
+    When I run the :create client command with:
+      | f | rsys_clf_invalid_values.yaml |
+    Then the step should fail
+
+  # @author gkarager@redhat.com
+  @admin
+  @destructive
+  Scenario Outline: Forward logs to remote-syslog
+    Given the master version >= "4.6"
+    Given I switch to the first user
+    And I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj clipboard
+    Given I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+
+    Given I obtain test data file "logging/clusterlogging/example.yaml"
```

Comment:
@gkarager I think we can use `example_indexmanagement.yaml ` as `the master version >= "4.6"` is added.